config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
   post "/graphql", to: "graphql#execute"
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+  if Rails.env.development?
+    mount GraphiQL::Rails::Engine, at: 'graphiql', graphql_path: "graphql#execute"
+    end
+    
+    post "/graphql", to: "graphql#execute"
```

Comment:
this allows us to hit graphiql when in development locally

Reply:
This looks good, and matches the tutorial I have been following.